eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -45,8 +45,19 @@ jobs:
 
       - template: ../steps/common.yml
 
+      - script: |
+          node eng/tools/generate-dev-builds.js "$(Build.BuildNumber)"
+        condition: eq(variables['DevBuildRun'],'true')
+        displayName: "Update package versions for dev build"
+
+      - script: |
```

Comment:
We should try and do this step for any scheduled build on the internal teams project so something like. 

```
${{if and(variables['Build.Reason'], 'Schedule', eq(variables['System.TeamProject'], 'internal'))}}
```

Reply:
It is probably worth doing something similar to what I did in .NET at https://github.com/Azure/azure-sdk-for-net/blob/master/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L15 to set the DailyDevRun property. For JS I think we only want to set it for scheduled internal builds.

Reply:
done